src/index.js

```diff
@@ -116,6 +116,22 @@ class Mongo {
     return user;
   }
 
+  async findUserByEmailVerificationToken(token: string): Promise<?UserObjectType> {
+    const user = await this.collection.findOne({ 'services.email.verificationTokens.token': token });
```

Comment:
I think all the find functions needs to have a projection parameter to limit which fields are selected.